src/DistributedMetadata/CatalogService.cpp

```diff
@@ -658,6 +678,7 @@ void CatalogService::mergeCatalog(const NodePtr & node, TableContainerPerNode sn
             auto iter_by_name = indexed_by_name.find(std::make_pair(p.second->database, p.second->name));
             assert(iter_by_name != indexed_by_name.end());
 
+            /// Deleted Store remove from `storages`
```

Comment:
Let's remove this comment because it is confusing

Reply:
fixed